### PR TITLE
Add `test-lint` Makefile rule

### PR DESCRIPTION
**Pull request by mertyildiran**
For local linting.

### CODE REVIEW

**Comment by mertyildiran** [comment]
Split from https://github.com/up9inc/mizu/pull/749

**Review by gadotroee — APPROVED** [review]
I understand you took it from the place you did just to make it available before.

**Comment by gadotroee** [comment]
> I understand you took it from the place you did just to make it available before.

nvm, I see now the message you wrote..

File: Makefile [hunk 1]
```diff
@@ -83,6 +83,15 @@ clean-cli:  ## Clean CLI.
 clean-docker:
 	@(echo "DOCKER cleanup - NOT IMPLEMENTED YET " )
 
+test-lint:
```
Comment (IgorGov) [inline Makefile:86]: i would call it "go-lint" as it not running FE linter, and not to confuse it with "tests"

Comment (mertyildiran) [reply Makefile:86]: oh we can run the ESLint as well. `test-` is the prefix for the rules that are related to testing. Linting is a test as well.

**Pull request merged**